### PR TITLE
feat(TransferFunctionWidget): use itk-viewer-transfer-function-editor

### PR DESCRIPTION
**Pull request by PaulHax**
- [x] Depends on https://github.com/Kitware/itk-vtk-viewer/pull/545

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,32 +20,36 @@
...
-    )
-
-    const uiMachineOptions = {
-      href: new URL(
-        '/src/bootstrapUIMachineOptions.js',
+      "
+    ></div>
+    <script
+      type="text/javascript"
+      src="http://localhost:8082/itkVtkViewer.js"
```
Comment (thewtex) [inline index.html:27]: -> deployed

File: src/Images/TransferFunctionWidget.jsx [hunk 2]
```diff
@@ -1,253 +1,24 @@
...
-      state.context.images.transferFunctionManipulator.levelGet = levelGet
-      const levelSet = () => {
-        const gaussians = transferFunctionWidget.getGaussians()
-        const newGaussians = gaussians.slice()
-        const name = state.context.images.selectedName
-        const component = state.context.images.selectedComponent
-        send({
-          type: 'IMAGE_PIECEWISE_FUNCTION_GAUSSIANS_CHANGED',
-          data: { name, component, gaussians: newGaussians }
-        })
```
Comment (thewtex) [inline src/Images/TransferFunctionWidget.jsx:151]: hurrah for simplification!

File: src/Images/createTransferFunctionEditor.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    data: {
+      name,
+      component,
+      range,
+      nodes
+    }
+  })
+}
+
+const vtkPiecewiseGaussianWidgetFacade = (tfEditor, context) => {
```
Comment (thewtex) [inline src/Images/createTransferFunctionEditor.js:52]: :student:

**Review by thewtex — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (PaulHax) [reply index.html]: Fixed

File: src/Images/TransferFunctionWidget.jsx (hunk 2 above)
Comment (PaulHax) [reply src/Images/TransferFunctionWidget.jsx:151]: The mess got swept into another file =)

**Review by thewtex — APPROVED** [review]

**Pull request merged**